### PR TITLE
manjarowsl: Add version 20211101

### PR DESCRIPTION
**Pull request by sitiom**
Reopening from #6089 because I accidentally deleted my fork.

Manifest derived from [archwsl](https://github.com/lukesampson/scoop-extras/blob/master/bucket/archwsl.json) and #5224.

### CODE REVIEW

File: bucket/manjarowsl.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        "   $installable = $false",
+        "}",
+        "if ($null -eq (Get-Command 'wslconfig' -ErrorAction SilentlyContinue)) {",
+        "    warn 'WSL appears not to be enabled!'",
+        "    warn 'Run ''Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux'''",
+        "    warn 'from an elevated PowerShell. Restart your computer when prompted and reinstall ManjaroWSL.'",
+        "    $installable = $false",
+        "}",
+        "if ($installable) {",
+        "    $installable = !(Invoke-ExternalCommand \"$dir\\Manjaro.exe\" 'isregd')",
```
Comment (rashil2000) [inline bucket/manjarowsl.json:31]: Put this line in the next if block itself.

File: bucket/manjarowsl.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        "}",
+        "if ($installable) {",
+        "    warn 'DO NOT KILL THE PROCESS. Installation is still running!'",
+        "    Copy-Item \"$dir\\Manjaro.exe\" \"$persist_dir\\data\\Manjaro.exe\"",
+        "    $res = Invoke-ExternalCommand \"$persist_dir\\data\\Manjaro.exe\" 'install', \"$dir\\rootfs.tar.gz\"",
+        "    Remove-Item \"$persist_dir\\data\\Manjaro.exe\" -Force",
+        "    if(!$res) { error 'ManjaroWsl installation failed!'; return }",
+        "}",
+        "Remove-Item \"$dir\\rootfs.tar.gz\" -Force"
```
Comment (rashil2000) [inline bucket/manjarowsl.json:40]: Shouldn't this line be in the above if block too?

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/manjarowsl.json (hunk 2 above)
Comment (sitiom) [reply bucket/manjarowsl.json:40]: No, I don't think it should be. There is a check in the first if block that changes `$installable` (also just copied that from #5224 so it should be suggested there too).

But I do think there is a better way to write that.

**Comment by rashil2000** [comment]
There are certain differences in the `post_install` of ArchWSL and your manifest (towards the end). Why is that?

**Comment by rashil2000** [comment]
I see that #5224 hasn't been merged yet. Let me review that first.

**Review by rashil2000 — APPROVED** [review]

**Comment by rashil2000** [comment]
@sitiom The checkver is failing (note that the version string is different from ArchWSL)

**Comment by sitiom** [comment]
> @sitiom The checkver is failing (note that the version string is different from ArchWSL)

My bad, didn't push the latest version earlier

**Review by issaclin32 — APPROVED** [review]
Tested. LGTM.

**Pull request merged**